specification/translation/Azure.AI.DocumentTranslation/client.tsp

```diff
@@ -3,3 +3,45 @@
  * Add readme and sample
  */
 import "./main.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.ClientGenerator.Core;
+using TypeSpec.Versioning;
+
+@TypeSpec.Versioning.useDependency(DocumentTranslation.Versions.v1_0)
+namespace ClientCustomizations;
+
+@added(DocumentTranslation.Versions.v1_0)
```

Comment:
If v1.0 is your baseline api version, you only need to add newer changes. So please remove this decorator